pino.js

```diff
@@ -203,7 +209,12 @@ Object.defineProperty(Pino.prototype, 'levelVal', {
 })
 
 Pino.prototype._setLevel = function _setLevel (level) {
-  if (typeof level === 'number') { level = this.levels.labels[level] }
+  if (typeof level === 'number') {
+    if (!isFinite(level)) {
+      throw new Error('unknown level ' + level)
```

Comment:
i used the same style as you used; the same construct is in the next if block; 
do you want to make this change in both places since i am already there ?